src/views/Dashboard/Customer/Home/index.vue

```diff
@@ -323,12 +323,12 @@ export default {
           )
         })
   
+        this.isLoadingPayments = false
         if(!this.paymentHistory.length) {
           this.titlePaymentWording = "You haven't made any order yet"
           return
         }
         this.titlePaymentWording = "Your recent payments"
-        this.isLoadingPayments = false
```

Comment:
Would be great if you change this logic into if else instead of using return inside if statement like this, so you don't have to displace this loading data

Reply:
updated